packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -171,33 +191,81 @@ export function compileBlueprint(
 			networking: blueprint.features?.networking ?? false,
 		},
 		run: async (playground: UniversalPHP) => {
+			const progressStatus = new Map<StepDefinition, Promise<void>>();
+			const progressPromises = new Map<
```

Comment:
`progress` as a word is already used for tracking the percentage 0 - 100% completion status. How about `stepPromises`?

Reply:
They've been renamed to `stepPromises` and `stepStatus`.